standards/ICRC-3/ICRC-3.did

```diff
@@ -0,0 +1,56 @@
+type Value = variant {
+    Blob : blob;
+    Text : text;
+    Nat : nat; // do we need this or can we just use Int?
+    Int : int;
+    Array : vec Value;
+    Map : vec record { text; Value };
+};
+
+type GetArchivesArgs = record {
+    // The last archive seen by the client.
+    // The Ledger will return archives coming
+    // after this one if set, otherwise it
+    // will return the first archives.
+    from : opt principal;
+};
+
+type GetArchivesResult = vec {
+    // The id of the archive
+    canister_id : principal;
+
+    // The first block in the archive
+    start : nat;
+
+    // The last block in the archive
+    end : nat;
+}
```

Comment:
i dont't think this is a valid candid type. is this supposed to be `vec record` or `record` instead?

Reply:
applies to readme as well

Reply:
Fixed with 8977901.